index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+   <!-- <h1>Hello World we are software developers</h1>-->
+   <main id="main-doc">
+
+	<nav id="navbar">
+		<header>JS Documentation</header>
+    <a href="#Introduction" class="nav-link">Introduction</a>
+    <a href="#What_you_should_already_know" class="nav-link">What you should already know</a>
+    <a href="#JavaScript_and_Java" class="nav-link">JavaScript and Java</a>
+    <a href="#Hello_world" class="nav-link">Hello world</a>
+    <a href="#Variables" class="nav-link">Variables</a>
+	
+	</nav>
```

Comment:
It's a better practice to have your nav wrapped in a `header` tag. Also kindly remember to use `ul` and `li` tags for lists.